Maths/PerfectSquare.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @function perfectSquare
+ * @description the square root of a number should be an integer
+ * @see [Perfect square](https://www.cuemath.com/algebra/perfect-squares/)
+ * @example Perfect square - 9, 16, 25
+ */
+
+
+/**
+ * @param {num} number 
```

Comment:
Merge these two doc comments and put them right above the `PerfectSquare` func

Reply:
Done